components/shared/Editor.tsx

```diff
@@ -337,6 +369,23 @@ export default function MockupEditor() {
     }
   }, [background]);
 
+  useEffect(() => {
+    if(customImg === ""){
+      setIsUrlFormat(true);
+    }
+    else if(customImg?.startsWith("http") || customImg?.startsWith("data:image/")){
+      const img = new Image();
+      img.src = customImg;
+      img.onload = () => {
+        setBackgroundImage(img);
+        setIsBackgroundLoaded(true);
```

Comment:
Are two useEffects are needed for making one functionality - setting a background.
Can we optimize this?